src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,7 +1,44 @@
 import React from 'react';
-
+import { Movie } from '../../types/Movie';
 import './MovieCard.scss';
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
+export const MovieCard: React.FC<Movie> = ({
+  title,
+  description = '',
+  imgUrl,
+  imdbUrl,
+  imdbId,
+}) => (
+  <div className="card" key={imdbId}>
```

Comment:
```suggestion
  <div className="card">
```
клас "card" мав би бути прямим нащадком класу "movies".
А в тебе зараз йде ul class="movies" > li > div class="card", тому картки виглядають не рівно.
Можеш цей div переробити на li, а на верхьому рівні прибрати li і мапити прямо компоненти MovieCard.
Або ж на верхньому рівні для li задати клас "card", а тут прибрати цей div і використати замість нього реакт фрагмент

Атрибут key тут не потрібен. Він вказується в самому методі map рівнем вище

<img width="1135" alt="image" src="https://user-images.githubusercontent.com/66570142/158578537-f154fce4-3ca4-44d9-a494-d021fab7db22.png">